pkg/agent/controller/traceflow/traceflow_controller.go

```diff
@@ -60,7 +60,8 @@ const (
 	defaultWorkers = 4
 	// Seconds delay before injecting packet into OVS. The time of different nodes may not be completely
 	// synchronized, which requires a delay before inject packet.
-	injectPacketDelay = 5
+	injectPacketDelay      = 2000
```

Comment:
Please update the comment which says the unit is seconds.

Reply:
And perhaps "Delay in milliseconds before ..."

Reply:
Thanks. Will fix here.